docker-example/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN python3.11 -m venv $VIRTUAL_ENV
 RUN pip install -r requirements.txt
 
 # Deploy templates and prepare app
-RUN reflex init
+RUN reflex init --template blank
```

Comment:
we actually probably don't need this in the dockerfile, because the app should already exist when the container is being built... not sure if it hurts anything though.

Reply:
Good point - I'll remove this